### PR TITLE
OSDOCS-10591 OCP 4.13.42 RNs

### PR DESCRIPTION
**Pull request by cbippley**
Version(s): 4.13

Issue: [OSDOCS-10591](https://issues.redhat.com/browse/OSDOCS-10591)

Link to docs preview: [Preview](https://76267--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-13-release-notes.html#ocp-4-13-42)

QE review: Not needed

Additional information: Expected merge date is May 23, errata links will not work until then.

### CODE REVIEW

**Review by mletalie — COMMENTED** [review]
/label peer-review-in-progress

File: release_notes/ocp-4-13-release-notes.adoc [hunk 1]
```diff
@@ -4108,3 +4108,52 @@ $ oc adm release info 4.13.41 --pullspecs
...
+
+* Previously, some container processes created by using the `exec` command persisted even when CRI-O stopped the container. Consequently, lingering processes led to tracking issues, causing process leaks and defunct statuses. With this release, CRI-O tracks the `exec` calls processed for a container and ensures that the processes created as part of the `exec` calls are terminated when the container is stopped.(link:https://issues.redhat.com/browse/OCPBUGS-33174[*OCPBUGS-33174*])
+
+* Previously, Terraform would create the compute server group with the policy set for the control plane. As a consequence, the 'serverGroupPolicy' property of the install-config.yaml file was ignored for the compute server group. With this release, the server group policy set in the install-config.yaml file for the compute MachinePool is correctly applied at install-time in the Terraform flow. (link:https://issues.redhat.com/browse/OCPBUGS-33062[*OCPBUGS-33062*])
```
Comment (mletalie) [inline release_notes/ocp-4-13-release-notes.adoc:4141]: //server group difference
installation?
```suggestion
* Previously, Terraform would create the compute server group with the policy set for the control plane. As a consequence, the 'serverGroupPolicy' property of the install-config.yaml file was ignored for the compute server group. With this release, the server group policy set in the install-config.yaml file for the compute MachinePool is correctly applied during installation in the Terraform flow. (link:https://issues.redhat.com/browse/OCPBUGS-33062[*OCPBUGS-33062*])
```

Comment (mletalie) [reply release_notes/ocp-4-13-release-notes.adoc:4141]: Question: Are these referring to the same thing? 
"compute server group with the policy set" and "the server group policy set"
If so, should they be identified in the same way? Hopfully what I am asking makes sense :)

File: release_notes/ocp-4-13-release-notes.adoc [hunk 2]
```diff
@@ -4108,3 +4108,52 @@ $ oc adm release info 4.13.41 --pullspecs
...
+
+[source,terminal]
+----
+$ oc adm release info 4.13.42 --pullspecs
+----
+
+[id="ocp-4-13-42-bug-fixes"]
+==== Bug fixes
+
+* Previously, for the `cluster:capacity_cpu_cores:sum` metric, nodes with the`infra` role but not `master` role were not assigned a value of `infra` for the `label_node_role_kubernetes_io` label. With this update, nodes with the `infra` role but not `master` role are now correctly labeled as `infra` for this metric.(link:https://issues.redhat.com/browse/OCPBUGS-33581[*OCPBUGS-33581*])
```
Comment (mletalie) [inline release_notes/ocp-4-13-release-notes.adoc:4131]: Anyway to refer to "master" as something else (to abide by conscious language efforts)? 
If not, totally understand, just wanted to mention it.

File: release_notes/ocp-4-13-release-notes.adoc [hunk 3]
```diff
@@ -4108,3 +4108,52 @@ $ oc adm release info 4.13.41 --pullspecs
...
+
+* Previously, the load balancing algorithm did not differentiate between active and inactive services when determining weights, and it employed the `random` algorithm excessively in environments with many inactive services or environments routing backends with weight 0. This led to increased memory usage and a higher risk of excessive memory consumption. With this release, changes are made to optimize traffic direction towards active services only and prevent unnecessary use of the `random` algorithm with higher weights, reducing the potential for excessive memory consumption. (link:https://issues.redhat.com/browse/OCPBUGS-33448[*OCPBUGS-33448*])
```
[the remark this answers is not in the document]
Comment (mletalie) [reply release_notes/ocp-4-13-release-notes.adoc:4133]: Suggestion as a way to remove passive voice..

File: release_notes/ocp-4-13-release-notes.adoc [hunk 4]
```diff
@@ -4108,3 +4108,52 @@ $ oc adm release info 4.13.41 --pullspecs
...
+
+* Previously, timeout values larger than what Golang could parse were not properly validated. Consequently, timeout values larger than what HAProxy could parse caused issues with HAProxy. With this update, if the timeout specifies a values larger than what can be parsed, it is capped at the maximum that HAProxy can parse. As a result, issues are no longer caused for HAProxy. (link:https://issues.redhat.com/browse/OCPBUGS-33280[*OCPBUGS-33280*])
+
+* Previously, a kernel regression that was introduced on {product-title} 4.14.14 caused kernel issues, such as nodes crashing and rebooting, in nodes that mounted to CephFS storage. In this release, the regression issue is fixed so that the kernel regression issue no longer occurs. (link:https://issues.redhat.com/browse/OCPBUGS-33252[*OCPBUGS-33252*])
```
[the remark this answers is not in the document]
Comment (mletalie) [reply release_notes/ocp-4-13-release-notes.adoc:4137]: Suggestion to remove clause.

**Review by mletalie — COMMENTED** [review]
/label peer-review-done
/remove-label peer-review-in-progress
/remove-label peer-review-needed
/lgtm

File: release_notes/ocp-4-13-release-notes.adoc (hunk 2 above)
Comment (cbippley) [reply release_notes/ocp-4-13-release-notes.adoc:4131]: Checking with the dev.

File: release_notes/ocp-4-13-release-notes.adoc (hunk 1 above)
Comment (cbippley) [reply release_notes/ocp-4-13-release-notes.adoc:4141]: Checking with dev

File: release_notes/ocp-4-13-release-notes.adoc (hunk 2 above)
Comment (cbippley) [reply release_notes/ocp-4-13-release-notes.adoc:4131]: Looks like this is the language we have to use for now. The style guide says it's okay as long as it's not paired with 'slave.'

File: release_notes/ocp-4-13-release-notes.adoc (hunk 1 above)
Comment (cbippley) [reply release_notes/ocp-4-13-release-notes.adoc]: Made a change for clarity.

**Review by aireilly — APPROVED** [review]
/lgtm

**Pull request merged**